addons/point_of_sale/tests/test_point_of_sale_flow.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+
+import openerp
+from openerp import fields
+from openerp.addons.point_of_sale.tests.common import TestPointOfSaleCommon
+
+@openerp.tests.common.at_install(False)
+@openerp.tests.common.post_install(True)
+class TestPointOfSaleFlow(TestPointOfSaleCommon):
+
+    def test_create_account_move_line(self):
+        # This test will check that the correct journal entries are created when a product in real time valuation
+        # is sold in a company using anglo-saxon
+        self.pos_config.open_session_cb()
+
+        # I create a PoS order with 1 unit of New product at 450 EUR
+        self.pos_order_pos0 = self.PosOrder.create({
+            'company_id': self.company.id,
+            'pricelist_id': self.company.partner_id.property_product_pricelist.id,
+            'lines': [(0, 0, {
+                'name': "OL/0001",
+                'product_id': self.product.id,
+                'price_unit': 450,
+                'discount': 0.0,
+                'qty': 1.0
+            })]
+        })
+
+        # I make a payment to fully pay the order
+        context_make_payment = {"active_ids": [self.pos_order_pos0.id], "active_id": self.pos_order_pos0.id}
+        self.pos_make_payment_0 = self.PosMakePayment.with_context(context_make_payment).create({
+            'amount': 450.0
+        })
+
+        # I click on the validate button to register the payment.
+        context_payment = {'active_id': self.pos_order_pos0.id}
+        self.pos_make_payment_0.with_context(context_payment).check()
+
+        # I check that the order is marked as paid
+        self.assertEqual(self.pos_order_pos0.state, 'paid', 'Order should be in paid state.')
+
+        # I generate the journal entries
+        self.pos_order_pos0._create_account_move_line()
+
+        local_context = {'force_company': self.company.id, 'company_id': self.company.id}
+
+        # I test that the generated journal entries are correct.
+        account_output = self.category.property_stock_account_output_categ_id
+        expense_account = self.category.property_account_expense_categ_id
+        aml = self.pos_order_pos0.account_move.line_ids
+        aml_output = aml.filtered(lambda l: l.account_id.id == account_output.id)
+        aml_expense = aml.filtered(lambda l: l.account_id.id == expense_account.id)
+        self.assertEqual(aml_output.debit, self.product.standard_price)
+        self.assertEqual(aml_expense.credit, self.product.standard_price)
```

Comment:
Ideally, `assertEqual()` should have a third parameter with a description of the problem, e.g. "Cost of Good Sold entry missing or mismatching"